# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1458318323 14400
#      Fri Mar 18 12:25:23 2016 -0400
# Node ID 19e952b828196f8651589656265d9c1b02b6099a
# Parent  4f3e63d75f331f481207242b71d4d74e5f7eec7c
delete unnecessary template Array instantiation file

* libinterp/template-inst/Array-os.cc: Delete.
* libinterp/template-inst/module.mk: Update.

diff --git a/libinterp/template-inst/Array-os.cc b/libinterp/template-inst/Array-os.cc
deleted file mode 100644
--- a/libinterp/template-inst/Array-os.cc
+++ /dev/null
@@ -1,38 +0,0 @@
-/*
-
-Copyright (C) 1996-2015 John W. Eaton
-
-This file is part of Octave.
-
-Octave is free software; you can redistribute it and/or modify it
-under the terms of the GNU General Public License as published by the
-Free Software Foundation; either version 3 of the License, or (at your
-option) any later version.
-
-Octave is distributed in the hope that it will be useful, but WITHOUT
-ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
-FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
-for more details.
-
-You should have received a copy of the GNU General Public License
-along with Octave; see the file COPYING.  If not, see
-<http://www.gnu.org/licenses/>.
-
-*/
-
-// Instantiate Arrays of octave_stream objects.
-
-#ifdef HAVE_CONFIG_H
-#  include "config.h"
-#endif
-
-#include "Array.h"
-#include "Array.cc"
-
-extern template class OCTAVE_API Array<bool>;
-extern template class OCTAVE_API Array<octave_idx_type>;
-
-#include "oct-stream.h"
-
-NO_INSTANTIATE_ARRAY_SORT (octave_stream);
-INSTANTIATE_ARRAY (octave_stream, OCTINTERP_API);
diff --git a/libinterp/template-inst/module.mk b/libinterp/template-inst/module.mk
--- a/libinterp/template-inst/module.mk
+++ b/libinterp/template-inst/module.mk
@@ -1,4 +1,3 @@
 TEMPLATE_INST_SRC = \
-  libinterp/template-inst/Array-os.cc \
   libinterp/template-inst/Array-tc.cc \
   libinterp/template-inst/Array-jit.cc
